snow/validators/validator.go

```diff
@@ -26,6 +26,19 @@ type Validator struct {
 // a validator of the Avalanche Network for the output of GetValidator.
 type GetValidatorOutput struct {
 	NodeID    ids.NodeID
-	PublicKey *bls.PublicKey
+	PublicKey PublicKey
 	Weight    uint64
 }
+
+type PublicKey struct {
+	*bls.PublicKey
```

Comment:
Why are we embedding this?

Reply:
I can remove the embed. I think I originally did this because because `GetBytes()` was originally named `Serialize()` existing code that looks like `publicKey.Serialize()` would not need a change but in hindsight it doesn't make the diff much smaller and I ended up renaming it anyways.